src/embed.ts

```diff
@@ -299,7 +299,7 @@ class Torus {
     if (!this.isInitialized) throw new Error("Torus is not initialized");
     const windowId = getWindowId();
     this.communicationProvider._handleWindow(windowId);
-    const topupResponse = await this.communicationProvider.request<boolean>({
+    const topupResponse = await this.communicationProvider.request<unknown, boolean>({
```

Comment:
This was straight up laziness on my part. It should actually be the type of `{ provider, params, windowId }`.

Reply:
I'll make a PR for this one. Thanks